geoh5py/objects/block_model.py

```diff
@@ -43,21 +51,42 @@ class BlockModel(GridObject):
     _attribute_map = GridObject._attribute_map.copy()
     _attribute_map.update({"Origin": "origin", "Rotation": "rotation"})
 
-    def __init__(self, object_type: ObjectType, **kwargs):
-        self._origin: np.ndarray = np.zeros(3)
-        self._rotation: float = 0.0
-        self._u_cell_delimiters: np.ndarray | None = None
-        self._v_cell_delimiters: np.ndarray | None = None
-        self._z_cell_delimiters: np.ndarray | None = None
-
-        super().__init__(object_type, **kwargs)
+    def __init__(
+        self,
+        object_type: ObjectType,
```

Comment:
Just need an explanation for my understanding:

why object_type?
I understand Workspace, or parent (which disappear)
But object_type is strange. 

I guess, as user, we can pass a workspace or a parent, and it creates an Object type?

Reply:
It's just the pattern that was originally used for all entities (groups, objects, data). First, the type is created and registered to the workspace, which validates that it is unique. For example data can share their types or create new ones, while objects and groups have fixed ones. Then the type is given to the entity as an attribute (already validated)